composer.json

```diff
@@ -23,6 +23,7 @@
         "psr/container": "^1.0",
         "psr/log": "~1.0",
         "psr/simple-cache": "^1.0",
+        "fig/link-util": "^1.0",
```

Comment:
I think we should require `psr/link` directly too? Because `fig/link-util` is just an implementation, even though it does add a transitive dependency on `psr/link`.